src/tools/rbd/action/Status.cc

```diff
@@ -190,7 +192,75 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
     }
 
     if (!image_cache_str.empty()) {
+      JSONFormattable f;
+      JSONParser p;
+      bool success = p.parse(image_cache_str.c_str(), image_cache_str.size());
+      if (!success) {
         std::cout << "Image cache state: " << image_cache_str << std::endl;
+      } else {
+        decode_json_obj(f, &p);
+        /*
+        utime_t update_time;
+        std::istringstream iss_update_time(f["update_time"]);
+        iss_update_time >> update_time;
```

Comment:
@idryomov A low-level question, I don't know how to get time from json. I tried some method but failed. Please give me some guidance, thanks.